lib/ansible/plugins/connection/netconf.py

```diff
@@ -79,7 +131,7 @@ def _connect(self):
 
         if not network_os:
             for cls in netconf_loader.all(class_only=True):
-                network_os = cls.guess_network_os(self)
+                Network_os = cls.guess_network_os(self)
```

Comment:
Is this a typo? if not `network_os` should be replaced by `Network_os` at other places as well.

Reply:
that is typo